packages/solana/src/client.ts

```diff
@@ -244,11 +244,17 @@ export class Web3Modal extends Web3ModalScaffold {
       this.syncNetwork(chainImages)
     })
 
-    NetworkController.subscribeKey('caipNetwork', () => {
+    NetworkController.subscribeKey('caipNetwork', (newCaipChain: CaipNetwork | undefined) => {
```

Comment:
Just for aligning naming, could we make it `newCaipNetwork?`
```suggestion
    NetworkController.subscribeKey('caipNetwork', (newCaipNetwork: CaipNetwork | undefined) => {
```

Reply:
And I guess you don't have to set type definitions since `subscribeKey` should already been set the type of the parameter.
```suggestion
    NetworkController.subscribeKey('caipNetwork', (newCaipChain) => {
```